### ISSUE TITLE
Can't access mod's setings

### ISSUE THREAD

**Issue by FabulousAzzzzzz** [Ferlern/extended-UI-v7#13]
I do not know if I am just plain dumb, but when checking settings/graphics, i see no options to change the mod settings

**Comment by lifestarse** [comment]
> I do not know if I am just plain dumb, but when checking settings/graphics, i see no options to change the mod settings

Yes, when you press the reset defauls button or flips the screen from landscape to portrait. That mod menu may disappear

**Comment by Ferlern** [comment]
Try to go into the settings immediately after starting the game. They are not there initially or they disappear for one of the reasons described by lifestarse?

**Comment by FabulousAzzzzzz** [comment]
> Try to go into the settings immediately after starting the game. They are not there initially or they disappear for one of the reasons described by lifestarse?

Sorry for not answering before, I did not see the message, after re-enabling the mod and restarting the game, the settings menu just appeared, but when resetting to default, it disappears, I think I just missed that detail, oops.

**Comment by Ferlern** [comment]
I know about this problem, but when I was developing the mod I did not figure out how to solve it

**Comment by lifestarse** [comment]
I think You can move Extended settings from graphics to main settings and
it's not disappear anymore.

**Comment by Ferlern** [comment]
Should not be a problem anymore, settings are moved to the main settings menu
